src/components/DevOpsButton/DevOpsButton.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { colorBlue, colorDarkBlue } from '../../styles/designTokens';
+import RoleButton from '../RoleButton';
+
+const StyledRoleButton = styled(RoleButton)`
+  background: ${colorBlue};
+  border-bottom-color: ${colorDarkBlue};
+`;
+
+function DevOpsButton() {
```

Comment:
What is the point of creating separate component for Roles buttons? Can't we just go in homepage with:
```
<StyledRoleButton to="/frontend" color="xxxx">FrontEnd</StyledRoleButton>
<StyledRoleButton to="/backend" color="xxxx">Backend</StyledRoleButton>
<StyledRoleButton to="/devops" color="xxxx">DevOps</StyledRoleButton>
```

Or if you really want to keep the RoleButton API simple, jutro make colour condition inside `StyledRoleButton`

Reply:
We can also separate the service `generateColors`, and make it more abstract to handle such scenarios.

Reply:
Also it should be called just `RoleButton`.